ru.json

```
@@ -0,0 +1,181 @@
+		"settings": "Настройки",
+		"reports": "Жалобы",
+		"moderation": "Модерация",
+		"notifications": "Уведомления",
+		"about": "О проекте",
+		"following": "Подписки",
+		"all": "Всё"
+	},
+	"generic": {
+		"labels": {
+			"license": "Лицензия",
+			"project_id": "ID проекта",
+			"project_status": "Статус проекта"
+		},
+		"actions": {
+			"edit": "Редактировать",
+			"follow": "Подписаться",
+			"unfollow": "Отписаться",
+			"report": "Пожаловаться",
+			"new_project": "Новый проект",
+			"download": "Скачать",
+			"save": "Сохранить",
+			"cancel": "Отмена",
+			"discard": "Сбросить"
+		},
+		"external": {
+			"github_profile": "Профиль на GitHub",
+			"discord": "Discord",
+			"github": "GitHub",
+			"issues": "Баг-трекер",
+			"source": "Исходный код",
+			"wiki": "Вики",
+			"patreon": "Patreon",
+			"paypal": "PayPal",
+			"buy_me_a_coffee": "Buy Me a Coffee",
+			"github_sponsors": "GitHub Sponsors",
+			"donate": "Пожертвовать",
+			"kofi": "Ko-Fi"
+		},
+		"environments": {
+			"label": "Окружение",
+			"server_side": "Сервер",
+			"client_side": "Клиент",
+			"values": {
+				"required": "Требуется",
+				"unsupported": "Не поддерживается",
+				"optional": "Необязательно",
+				"unknown": "Неизвестно"
+			}
+		},
+		"byline": "от <u>{author}</u>"
+	},
+	"project": {
+		"types": {
+			"mod": {
+				"singular": "Мод",
+				"plural": "Моды",
+				"search": "Поиск модов..."
+			},
+			"modpack": {
+				"singular": "Сборка модов",
+				"plural": "Сборки модов",
+				"search": "Поиск сборок модов..."
+			}
+		},
+		"sidebar_headings": {
+			"external_resources": "Внешние ресурсы",
+			"featured_versions": "Рекомендуемые версии",
+			"project_members": "Участники проекта",
+			"technical_information": "Техническая информация"
+		}
+	},
+	"status": {
+		"approved": "Утверждён",
+		"rejected": "Отклонён",
+		"draft": "Черновик",
+		"unlisted": "Скрыт",
+		"processing": "На рассмотрении",
+		"unknown": "Неизвестно"
+	},
+	"release_channels": {
+		"release": "Релиз",
+		"beta": "Бета",
+		"alpha": "Альфа"
+	},
+	"roles": {
+		"developer": "Разработчик",
+		"admin": "Администратор",
+		"moderator": "Модератор"
+	},
+	"stats": {
+		"joined": "Зарегистрировался {ago}",
+		"notified": "Получено {ago}",
+		"downloads": "{downloads, plural, one {<b>#</b> скачивание} few {<b>#</b> скачивания} many {<b>#</b> скачиваний} other {<b>#</b> скачивания}}",
+		"followers_of_projects": "{followers, plural, one {<b>#</b> подписчик на проекты} few {<b>#</b> подписчика на проекты} mane {<b>#</b> подписчиков на проекты} other {<b>#</b> подписчика на проекты}}",
+		"followers": "{followers, plural, one {<b>#</b> подписчик} few {<b>#</b> подписчика} many {<b>#</b> подписчиков} other {<b>#</b> подписчика}}",
+		"user_id": "ID пользователя: {id}",
+		"created": "Создан {ago}",
+		"updated": "Обновлён {ago}"
+	},
+	"tags": {
+		"technology": "Технологии",
+		"adventure": "Приключения",
+		"magic": "Магия",
+		"utility": "Утилиты",
+		"decoration": "Декоративные",
```

```suggestion
		"decoration": "Декор",
```
Как я и писал в Дискорде,  к этой категории относятся и косметические моды для клиента, поэтому `декорации` неуместно. Возможно, стоит рассмотреть вариант `косметика` (в Mod Manager так и переведено, кстати).